PypeS/pype.py

```diff
@@ -144,7 +166,7 @@ def ParseArguments(self):
                 self.AutoPipe = 0
             if '--nolog' in pypeArguments:
                 self.LogOn = 0
-            if '--query' in pypeArguments:
+            if '--query' in pypeArguments: #TODO: What on earth does this do?
```

Comment:
If I remember well 😄 , it was a facility to get all scripts from a package. Mainly done for PypePad. It's broken now, it should go for good.